BREAKING.md

```diff
@@ -40,6 +41,19 @@ The `SharedNumberSequence` and `SharedObjectSequence` have been deprecated and a
 
 Additionally, `useSyncedArray()` from `@fluid-experimental/react` has been removed, as it depended on the `SharedObjectArray`.
 
+### `IContainer` interface updated to complete 0.53 changes
+The breaking changes introduced in [`IContainer` interface updated to expose actively used `Container` public APIs](#IContainer-interface-updated-to-expose-actively-used-Container-public-APIs) have now been completed in 0.54. The following additions to the `IContainer` interface are no longer optional but rather mandatory:
+- `connectionState`
+- `connected`
+- `setAutoReconnect()` (**alpha**)
```

Comment:
update the notes here with info on the latest changes and intentions around the optional/alpha-ness

Reply:
Those are included and linked to with the notes in the earlier release

Reply:
Oh I see what you mean, will do